tests/ui/transmutability/references/reject_extension.stderr

```diff
@@ -0,0 +1,25 @@
+error[E0277]: `&Packed<Two>` cannot be safely transmuted into `&Packed<Four>`
+  --> $DIR/reject_extension.rs:48:37
+   |
+LL |     assert::is_transmutable::<&Src, &Dst>();
```

Comment:
I don't like that only the second parameter is `^^^^`, but that's a general issue with all of our error messages that I'll file a follow-up PR for. 

Otherwise, this error message LGTM.